packages/opds-web-client/src/state.ts

```diff
@@ -18,8 +18,8 @@ export interface State {
     package, but it's available for other apps if they need to build the state
     for server-side rendering. */
 export default function buildInitialState(
-  collectionUrl: string,
-  bookUrl: string
+  collectionUrl?: string,
+  bookUrl?: string
```

Comment:
For example, @adriana-alter , here you can imagine that the initial state could be built without a bookUrl - in fact unless the user is specifically requesting a /book/:bookUrl route, I believe there would be no bookUrl provided here, but it is marked as required. Which means that when you turn on strictNullChecks, I had to pass in a string at all times to this function.

Reply:
Makes sense; thank you!